src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements.mdx

```diff
@@ -9,12 +9,32 @@ redirects:
   - /docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements
 ---
 
-[New Relic's Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) can be [installed directly](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#install) on a server or VM, or through several [cloud platforms](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#cloud-platforms), such as GKE, EKS, AKS, or OpenShift. Each has a different compatibility with our integration.
+The [Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) is compatible with many different platforms including GKE, EKS, AKS, OpenShift, and more. Each has a different compatibility with our integration. You can find more information in this page.
+
+## Requirements [#reqs]
+
+The New Relic Kubernetes integration has the following requirements:
+
+* A New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
+* Linux distribution [compatible with New Relic infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure#operating-systems).
+* [`kube-state-metrics`](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics) running on the cluster.
+* When using CRI-O as the container runtime, the [processes](/attribute-dictionary/?event=ProcessSample) inside containers are not reported. Performance data is collected at the [container](/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data#container-data) level.
```

Comment:
IIRC this is true both for CRI-O and containerd, which are the two runtimes that are not deprecated at the moment.
Which is another way of saying that we only support this feature for deprecated runtimes.

I do not believe this feature is, in fact, being used a lot in kubernetes, so I'd get rid of this bullet point or at least demote it to a less prominent place.

Reply:
Removed.